.eslintrc

```diff
@@ -43,24 +44,35 @@
 			"error",
 			{ "ignoreVoid": true }
 		],
-		"@typescript-eslint/no-unused-vars": "warn",
 		"@typescript-eslint/explicit-function-return-type": "off",
+		"@typescript-eslint/explicit-module-boundary-types": "off",
```

Comment:
Explain rationale for these rule changes please.

Reply:
Usually disabling rules that throw a lot of false positives or are too pedantic. I'll have a closer look later.

Reply:
Also sorted them by name